charts/seed-controlplane/charts/dependency-watchdog/templates/rbac.yaml

```
@@ -0,0 +1,50 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: dependency-watchdog
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ toYaml .Values.dependencywatchdog.labels | indent 4 }}
+---
+apiVersion: {{ include "rbacversion" . }}
+kind: RoleBinding
+metadata:
+  name: system:dependency-watchdog-role-binding
```

```suggestion
  name: gardener.cloud:dependency-watchdog:role-binding
```
(Apply same naming scheme for other occurrences)